### PR TITLE
Delete legacy client

### PR DESCRIPTION
**Pull request by ericanderson**
Chains to #629

### CODE REVIEW

File: packages/foundry-sdk-generator/src/generate/betaClient/generatePackage.ts [hunk 1]
```diff
@@ -82,12 +72,7 @@ export async function generatePackage(
   };
 
   if (!options.beta) {
-    await generateClientSdkVersionOneDotOne(
-      ontology,
-      `typescript-sdk/${options.packageVersion} ${USER_AGENT}`,
-      hostFs,
-      packagePath,
-    );
+    throw new Error("Only beta is supported in this version");
```
Comment (ssanjay1) [inline packages/foundry-sdk-generator/src/generate/betaClient/generatePackage.ts:75]: Once we have the ability to just get our beta releases into dev console, we can just remove this flag right?

File: packages/cli.cmd.typescript/src/generate/handleGenerate.mts [hunk 2]
```diff
@@ -256,17 +254,15 @@ async function generateSourceFiles(
...
-    : generateClientSdkVersionOneDotOne)(
-      ontology,
-      getUserAgent(args.version),
-      fs,
-      args.outDir,
-      args.packageType,
-      args.ontologyApiNamespace,
-      args.apiNamespaceMap,
-    );
+  await generateClientSdkVersionTwoPointZero(
```
Comment (ssanjay1) [inline packages/cli.cmd.typescript/src/generate/handleGenerate.mts:257]: Not related to this, but looking at `handleGenerate` in this file, should we still have a beta flag for it?

**Review by ssanjay1 — APPROVED** [review]

File: packages/foundry-sdk-generator/src/generate/betaClient/generatePackage.ts (hunk 1 above)
Comment (ericanderson) [reply packages/foundry-sdk-generator/src/generate/betaClient/generatePackage.ts:75]: Correct, this is just hold over

File: packages/cli.cmd.typescript/src/generate/handleGenerate.mts (hunk 2 above)
Comment (ericanderson) [reply packages/cli.cmd.typescript/src/generate/handleGenerate.mts:257]: good call , we can clean that up now too

**Pull request merged**